### ISSUE TITLE
Show how to add a page title to routes

### ISSUE THREAD

**Issue by jenweber** [ember-learn/guides-source#1381]
Every route in an Ember app should have a Page Title, which helps for accessibility and SEO. In a Route's JavaScript file, you can do something like this:

```
  activate() { 
    document.title = 'Page Title Goes Here'; 
   },

  deactivate() {
     document.title = '';  
  }
```

To close this issue:
- Keep it short and sweet
- Add a code example to https://guides.emberjs.com/release/accessibility/page-template-considerations/
- Explain that this is the simplest approach that is a great first step for most apps. There are addons available that help do this in a more dynamic and maintainable way
- Note that the example above does not work with Fastboot apps
- Show how to test the page title (Ava will show an example in the comments below)

**Comment by jgwhite** [comment]
🤚 I’d like to tackle this

**Comment by sharpshark28** [comment]
Could have an example for how to write a simple test.

```
await visit('my-page');
assert.equal(document.title, 'Page Title Goes Here');
```

**Comment by locks** [comment]
Related PR is merged. Thanks!
